create-version-tags.py

```diff
@@ -0,0 +1,25 @@
+# Used by cloudbuild.yaml to tag images with correct versions based on git version tags.
+# This script assumes that there exists a file named _FULL_TAG in the container workspace
+# already, which contains the latest version number from git.
+try:
+    with open("_FULL_TAG") as f:
+        version = f.read()
+except FileNotFoundError:
+    print("Please run git describe --abbrev=0 --tags > _FULL_TAG first.")
+    quit()
```

Comment:
Consider using sys.exit() since it's always available but exit() and quit() are only available if the site module is imported